packages/pageheader/stories/helpers/PageHeaderOverflowMenu.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { SecondaryButton } from "../../../button";
+import { DropdownMenu, DropdownMenuItem } from "../../../dropdowMenu";
+import { SystemIcons } from "../../../icons/dist/system-icons-enum";
+import Dropdownable, {
+  Direction
+} from "../../../dropdownable/components/Dropdownable";
+
+class PageHeaderOverflowMenu extends React.PureComponent<
+  {},
+  { open: boolean }
+> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false
+    };
+
+    this.handleClose = this.handleClose.bind(this);
```

Comment:
you could evade the need to bind in typescript with instance properties. 

```
  handleClose = () => {
    this.setState({
      open: false
    });
  }
```

(just close that in case you already knew and picked the other way for a reason)